st2reactor/st2reactor/rules/worker.py

```diff
@@ -62,6 +63,12 @@ def process(self, pre_ack_response):
         if not trigger_instance:
             raise ValueError('No trigger_instance provided for processing.')
 
+        get_driver().inc_counter(
+            format_metrics_key(
+                key='trigger.%s' % trigger_instance.trigger
```

Comment:
Minor thing - usually it's safer to wrap string formatting arguments inside parenthesis in case we ever decide to add more format string arguments.